tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -2164,6 +2190,117 @@ static uint8_t padv_create_sync(const void *cmd, uint16_t cmd_len,
 
 	return BTP_STATUS_VAL(err);
 }
+
+#if defined(CONFIG_BT_PER_ADV_SYNC_TRANSFER_SENDER)
+static uint8_t padv_sync_transfer_set_info(const void *cmd, uint16_t cmd_len,
+					   void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_gap_padv_sync_transfer_set_info_cmd *cp = cmd;
+	struct bt_le_ext_adv *ext_adv = tester_gap_ext_adv_get(0);
```

Comment:
Since we support multiple advertising sets, perhaps this should do a lookup?

Reply:
I was wondering about this too, but PA implementation is always using set 0 (PADV and PAST predates multi adv support in BTP)

Reply:
Not strongly opposed to keeping as is, if PADV doesn't support it anyways, but arguably something someone should "fix" at some point (I am not aware of any test requiring multiple PADV)